installer/install.iss.in

```diff
@@ -7,8 +7,9 @@
 #else
 #define APP_VERSION   '%APPVERSION%'
 #endif
-#define APP_URL       'http://msysgit.github.io/'
-#define APP_BUILTINS  'etc\fileList-builtins.txt'
+#define MINGW_BITNESS '%MINGW_BITNESS%'
+#define APP_URL       'http://git-for-windows.github.io/'
+#define APP_BUILTINS  'share\git\builtins.txt'

```

Comment:
FYI: I did not test the behavior if this file is present. In my testing, I always accepted the printed out error message that the file is missing.